### PR TITLE
Nacl

### PR DESCRIPTION
**Pull request by OmarElawady**
Implement nacl and encryption module.

### CODE REVIEW

File: jumpscale/data/encryption/exceptions.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from jumpscale.core.exceptions import JSException
+
+
+class CoudntVerifyAuthorError(JSException):
```
Comment (xmonader) [inline jumpscale/data/encryption/exceptions.py]: CouldntVerifyAuthorError

File: jumpscale/data/encryption/mnemonic.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
+import os
+import hashlib
+
+from .exceptions import FailedChecksumError
+
+
+wordlist = []
+
+with open("jumpscale/data/encryption/wordlist.txt", "r") as f:
```
Comment (xmonader) [inline jumpscale/data/encryption/mnemonic.py]: no hardcoded paths.

File: jumpscale/data/encryption/exceptions.py (hunk 1 above)
Comment (OmarElawady) [reply jumpscale/data/encryption/exceptions.py]: Removed. This exception is not needed.

File: jumpscale/data/encryption/mnemonic.py (hunk 2 above)
Comment (OmarElawady) [reply jumpscale/data/encryption/mnemonic.py]: Used os.path.dirname(__file__) to retrieve the module's directory instead of the path.

**Pull request merged**